Library/Formula/srmio.rb

```diff
@@ -15,7 +15,7 @@ class Srmio < Formula
 
   def install
     if build.head?
-      system "chmod u+x genautomake.sh"
+      chmod 1, "genautomake.sh"
```

Comment:
We could roll our own symbolic Ruby `chmod` function pretty easily: have it read the current permissions, do bitmask and/or/set operations against the input, and write the resulting permissions out using the octal `chmod` form.

But `chmod 0755` sounds fine too.